onset_maproom/calc.py

```diff
@@ -61,13 +102,50 @@ def onset_date(
     dry_spell_search,
     time_coord="T",
 ):
-    """Finds the first wet spell of wet_spell_length days
-    where cumulative rain exceeds wet_spell_thresh,
-    with at least min_wet_days count of wet days (greater than wet_thresh),
-    not followed by a dry spell of dry_spell_length days of dry days (not wet),
-    for the following dry_spell_search days
-    returns the time delta rom the first day of daily_rain
-    to the first wet day in that wet spell
+    """ Calculate onset date.
+
+        Finds the first wet spell of 'wet_spell_length' days where: 
+            Cumulative rain exceeds 'wet_spell_thresh',
+            With at least 'min_wet_days' count of wet days (greater than 'wet_thresh'),
+            Not followed by a dry spell of 'dry_spell_length' days of dry days (not wet),
+            For the following 'dry_spell_search' days
+    
+        Parameters
+        ----------
+        daily_rain : DataArray
+                    Array of daily rainfall values.
+        wet_thresh : int
+                    Rainfall threshold to determine wet day.
```

Comment:
precise if greater than or greater or equal

Reply:
I am confused by this comment.

Reply:
"precise" is Frenglish for "specify."

Reply:
Oops... I mean to be more specific than "determine" by giving the specific definition, which must be `daily_rain` greater than (or greater or equal... to be checked) `wet_thresh`
I think you did it so in other similar Parameter Description elsewhere. I think...

Reply:
Is something like this more what you are hoping for?

`Rainfall threshold to determine wet day if 'daily_rain' is greater than 'wet_thresh'.`

Reply:
Yes, I fixed that in one of the commits I had just pushed :)